config/application.rb

```diff
@@ -91,6 +91,16 @@ class Application < Rails::Application
         redirect_uri: 'sidekiq/auth/github/callback'
       }
 
+      # Test User Dashboard configuration
+      if Settings.test_user_dashboard.env == 'staging'
```

Comment:
Is this conditional necessary here if this is already on the [route](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/routes.rb#L424)?

Reply:
Hey @rileyanderson, I just removed that conditional. Yeah, I agree, even if the block is executed it doesn't really matter. Thanks.